app/views/api/_observation.xml.builder

```diff
@@ -30,6 +30,18 @@ xml.tag!(tag,
         xml_detailed_object(xml, :sequence, sequence)
       end
     end
+    xml.notes do
```

Comment:
Let's review this to see if one of @pellaea's suggested renderings makes more sense, e.g., 
```
<fields>
  <field>
    <key>Cap</key>
    <value>red</value>
  </field>
  etc.
</fields>
<notes>Other notes, like before.</notes>
```
Does that feel any more intuitive to you?

You know what would be really sweet?  Try this for kicks:
```
<notes>
  <cap>red</cap>
  etc.
  <other>Other notes.</other>
</notes>
```
Of course this requires that all field names be simple ascii alphanumeric strings, maybe with spaces replaced by underscores.  ... Yeah, blah, that really is begging for trouble, isn't it?  I'm just trying to imagine use-cases like accessing common herbarium fields like habitat, substrate, etc. It would be really cool to be able to refer to them directly like this:

observation.fields.substrate

as opposed to having to look through all the fields searching for the one you want.  Of course, in the JSON format this is a moot point, and that's probably how everyone is going to be using the API in the future, anyway...